composer.json

```diff
@@ -27,7 +27,7 @@
     "autoload": {
         "psr-4": {
             "Zumba\\": "src/",
```

Comment:
@cjsaylor As we talked in person, this was left intentionally to keep the BC with the existent namespace. I marked as deprecated on this version.